src/python/xml_parser-py.c

```diff
@@ -73,13 +67,16 @@ c_newpkgcb(cr_Package **pkg,
 
     if (result == Py_None) {
         *pkg = NULL;
-        data->py_pkg = NULL;
-        Py_DECREF(result);
```

Comment:
I understand the change on 76, less so the change on 77, is it because `Py_None`  is a singleton value and therfore the refcount doesn't need to be managed if `result` is `Py_None`?

Reply:
Never mind - I think it's because PyDict_SetItem increments the count of `result`, but we're done with it here, so (new) line 78 handles both codepaths.